lib/utils/BaseReporter.js

```diff
@@ -208,7 +208,21 @@ class BaseReporter extends events.EventEmitter {
     }
 
     /**
-     * Outut the given failures as a list
+     * Output the given passes as a list
+     */
+    listPasses () {
+        console.log()
```

Comment:
I am not sure that any output should be in BaseReporter

Reply:
I'm not disagreeing (or agreeing necessarily!) - but I created listPasses() from the pre-existing listFailures() - which has basically the same output statements? ie, listPasses works in the same way as listFailures (for consistency) - but obviously for passes not fails! Can you explain further why you feel this is wrong?

Reply:
>Can you explain further why you feel this is wrong?

I mixed v5\v4 reporter behaviour.

>but I created listPasses() from the pre-existing listFailures()

yeah I see now